src/Dfe.Spi.GiasAdapter.Infrastructure.GiasSoapApi/SoapException.cs

```diff
@@ -0,0 +1,13 @@
+namespace Dfe.Spi.GiasAdapter.Infrastructure.GiasSoapApi
+{
+    public class SoapException : GiasSoapApiException
+    {
+        public string FaultCode { get; }
+
+        public SoapException(string faultCode, string faultString)
+            : base(faultString)
+        {
+            FaultCode = faultCode;
+        }
+    }
+}
```

Comment:
Again, re: the WCF client.. don't exceptions like these already exist within WCF?

Reply:
As we are not / cannot use WCF, no point brining in reference jsut for exception (If it were there at all. Not sure how re-usable it would be)